### ISSUE TITLE
Windows resource files should be automatically generated for all modules

### ISSUE THREAD

**Issue by keithc-ca** [eclipse/openj9#382]
Currently there's the hand-crafted j9jit.rc file for JIT's dll information. The contents of that file were outdated (improved by #358), but it really ought to be automatically generated as is the case for (most?) other modules.

sourcetools/com.ibm.uma/com/ibm/j9/uma/platform/PlatformWindows.java seems to make a special case for j9jit.

**Comment by imrahul361** [comment]
can i work on it?

**Comment by DanHeidinga** [comment]
Sure! @imrahul361 we'd be glad to have the help here.

fyi @dnakamura in case this affects the cmake migration

**Comment by keithc-ca** [comment]
Yes, of course! You will need to familiarize yourself with [1], in particular the agreements you'll be asked to sign.

[1] https://github.com/eclipse/openj9/blob/master/CONTRIBUTING.md

**Comment by TheMarvelFan** [comment]
Hi, is anyone working on this issue?

**Comment by keithc-ca** [comment]
> Hi, is anyone working on this issue?

Not to my knowledge. It would be good to get this (quite old) issue addressed.

**Comment by TheMarvelFan** [comment]
> but it really ought to be automatically generated as is the case for (most?) other modules.

@keithc-ca Can you please elaborate what technique do the other modules use to be auto-generated?

**Comment by keithc-ca** [comment]
I was about to suggest that we just close this, given that it was originally about the UMA build system which we (mostly) no longer use.

The JIT shared library appears to still be special in the cmake build system. It turns out it might be as simple as adding call to `j9vm_add_copyright_data()` in `runtime/compiler/CMakeLists.txt`. Other libraries automatically get the `.rc` file via use of `j9vm_add_library()`.

**Comment by keithc-ca** [comment]
UMA builds are no longer used for Semeru builds - I think cmake builds work as expected: closing.
